hasura/graphql-engine
# use inflection to suggest relationship names (close #1665)

### Description



**CHANGES PREVIOUS BEHAVIOUR**
Auto-generated relationship names are changed
**Will have to call this out in the release notes if the tracking bit is being used *wrongly* for CI/CD, especially auto-track**
cc: @shahidhk @rikinsk 

Fixes issue #1665 

### Affected components 


- Console

### Related Issues



#1665 
### Solution and Design


Uses inflection API's
### Steps to test and verify



- Create two tables author and article
- Add foreign key from article to author
- Navigate to article and verify if the suggested relationship is `author` for `author_id -> id` FK
- Navigate to the schema page and click on the button `Track all relationtions` and `individual track buttons` and validate the relationship names

### Limitations, known bugs & workarounds
